extensions/skins/conversation_v1/Conversation.js

```diff
@@ -208,11 +208,22 @@ oppia.directive('conversationSkin', [function() {
       $scope.PANEL_TUTOR = 'tutor';
       $scope.PANEL_SUPPLEMENTAL = 'supplemental';
 
+      $scope.helpHtml = null;
+      $scope.continueToNextCardHtml = null;
+
       $scope.profilePicture = '/images/avatar/user_blue_72px.png';
       oppiaPlayerService.getUserProfileImage().then(function(result) {
         $scope.profilePicture = result;
       });
 
+      $scope.clearHelpHtml = function() {
+        $scope.helpHtml = null;
+      };
+
+      $scope.clearContinueToNextCardHtml = function() {
+        $scope.continueToNextCardHtml = null;
+      };
+
```

Comment:
Replaced with $scope.clearHelpCard() that does multiple things. The advantage of having a separate function is that we can change just that function if we decide we need to (like I am about to do in the upcoming commit, in fact) -- it makes conceptual sense for it to be its own thing.